core/logger/zap.go

```diff
@@ -168,14 +171,17 @@ func (l *zapLogger) Named(name string) Logger {
 func (l *zapLogger) NewRootLogger(lvl zapcore.Level) (Logger, error) {
 	newLogger := *l
 	newLogger.config.Level = zap.NewAtomicLevelAt(lvl)
-
+	consoleCore, errWriter, err := newConsoleCore(newLogger.config)
+	if err != nil {
+		return nil, err
+	}
 	cores := []zapcore.Core{
 		// The console core is what we want to be unique per root, so we spin a new one here
-		newConsoleCore(newLogger.config),
+		consoleCore,
 	}
 	extraCores, err := newCores(newLogger.config)
```

Comment:
should rename this as well - to just newDiskCore

Reply:
this might as well get removed, since we already have a `newdiskcore`, and perform the validation where this is used